brooklyn-server/camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ExternalConfigYamlTest.java

```diff
@@ -108,6 +110,34 @@ public void testExternalisedLocationConfigReferencedFromYaml() throws Exception
         assertEquals(Iterables.getOnlyElement( app.getLocations() ).config().get(MY_CONFIG_KEY), "myval");
     }
 
+    @Test
+    public void testExternalisedCatalogConfigReferencedFromYaml() throws Exception {
+
+        String yaml = Joiner.on("\n").join(
+                "brooklyn.catalog:",
+                "    id: osgi.test",
+                "    itemType: template",
+                "    version: 1.3",
+                "    description: CentOS 6.6 With GUI - 1.3",
+                "    displayName: CentOS 6.6",
+                "    iconUrl: classpath:///centos.png",
+                "    brooklyn.libraries:",
+                "    - $brooklyn:external(\"myprovider\", \"osgikey\")",
```

Comment:
Can you also test with the format:

    brooklyn.libraries: [ { name: symbolic-name, version: 1.0, url: http://... }, ... ]

See https://github.com/apache/incubator-brooklyn/blob/master/brooklyn-docs/guide/ops/catalog/index.md for what is supported in a library value.